Documentation/git-rebase.txt

```diff
@@ -388,8 +388,9 @@ See also INCOMPATIBLE OPTIONS below.
 	as the committer date. This implies --force-rebase.
 
 --ignore-date::
-	This flag is passed to 'git am' to change the author date
-	of the rebased commits (see linkgit:git-am[1]).
+--author-date-is-committer-date::
```

Comment:
This is now not quite correct for am based rebases.  `--author-date-is-committer-date` doesn't exist there, and I don't see this patch implementing it.  Maybe introducing that flag in a preparatory patch would be nice.